Entities/Items/Sponge/SpongeDryInAir.as

```diff
@@ -0,0 +1,26 @@
+#include "SpongeCommon.as";
+
+const int slow_frequency = 5; // don't run this script too often
+
+void onInit(CBlob@ this)
+{
+	this.getCurrentScript().tickFrequency = 30 * 2 * slow_frequency; // 0.5 "absorbs" per seconds, run only every 5 "absorbs"
+}
+
+void onTick(CBlob@ this)
+{
+	u8 absorbed = this.get_u8(ABSORBED_PROP);
```

Comment:
It would be nice if we still did accelerate this when it's on fire or on/near a block on fire.

Reply:
Do you mean accelerate drying when sponge is in inventory too?